### DISCUSSION TITLE
Imaginary Situation Problem! [Ente Auth]

### DISCUSSION THREAD

**Discussion by NeeL21YT** [ente-io/ente#2433] [General]
Imagine that you are not logged into Ente Auth on any of your devices. Additionally, your Google account requires 2FA for login, but you enabled email verification. How will you login in Ente Auth if the code is sent to your Google account instead of any SMS-based system? Essentially, you are locked out of both your Google account and your Ente Auth account!

Just thinking, i know everyone has multiple devices login just in case but still, had to let my overthinking do it's work lmao!

**Comment by ua741** [comment]
Please see https://help.ente.io/auth/faq/enteception/#enteception
